src/foam/u2/view/ReciprocalSearch.js

```diff
@@ -113,6 +131,31 @@ foam.CLASS({
           predicate$: this.data$
         });
       }
+    },
+    {
+      class: 'Boolean',
+      name: 'isLoading',
```

Comment:
You could also just set the count to Number.NaN when you don't know and have the countText say Loading... when it's that value.

Reply:
Resolved by 24246fe6f5c09eb6b84f47ee279e11d36cfc8205. I took Mike's approach instead since it shows "Loading..." while we wait for the table to populate as well. The count usually finishes before the real data loads so it's a bit weird to see the count show up before the data loads. By waiting for all async requests to finish, "Loading..." is shown until everything is visible, which makes more sense from a UX perspective, in my opinion.